src/renderers/shaders/ShaderChunk/clipping_planes_fragment.glsl

```diff
@@ -0,0 +1,11 @@
+#if NUM_USER_CLIPPING_PLANES > 0
+
+	for ( int i = 0; i < NUM_USER_CLIPPING_PLANES; ++ i ) {
+
+		vec4 plane = userClippingPlanes[ i ];
+		if ( dot( vViewPosition, plane.xyz ) > plane.w ) discard;
```

Comment:
I'd suggest adding a math function for common.glsl so we can be a little higher level in this test.

```
if( planePointBehind( plane, vViewPosition ) discard;
```

I guess I was thinking we could create a math library in common.glsl (or split it out to geometry.glsl eventually)  once so we could use higher level functions in our glsl code.